rasa/core/training/story_reader/yaml_story_reader.py

```diff
@@ -249,6 +261,21 @@ def _parse_action(self, step: Dict[Text, Any]) -> None:
 
         self._add_event(action_name, {})
 
+    def _parse_bot_message(self, step: Dict[Text, Any]) -> None:
+        bot_message = step.get(KEY_BOT_END_TO_END_MESSAGE, "")
+        # TODO: Replace with schema validation
+        if not bot_message:
+            common_utils.raise_warning(
+                f"Issue found in '{self.source_name}': \n"
+                f"Bot message cannot be empty. "
+                f"This story step will be skipped:\n"
+                f"{step}",
+                docs=DOCS_URL_STORIES,
+            )
+            return
+
+        self._add_event(bot_message, {"e2e_text": bot_message})
```

Comment:
Just making sure I get things right: after this, we will have `ActionExecuted` where `action_name` and `e2e_text` will be set to the bot message, right?

Reply:
yep 👍